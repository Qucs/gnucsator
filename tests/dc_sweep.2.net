options trace

Eqn:Eqn1 I1="1" Export="yes"
Idc:I _net0 gnd I="I1"
.SW:SW1 Sim="DC1" Type="lin" Param="I1" Start="10m" Stop="50m" Points="5"
.DC:DC1 Temp="26.85" reltol="0.001" abstol="1 pA" vntol="1 uV" saveOPs="no" MaxIter="150" saveAll="no" convHelper="none" Solver="CroutLU"
R:R1 _net0 gnd R="1"
