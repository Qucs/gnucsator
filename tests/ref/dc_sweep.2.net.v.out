
module main ();
parameter I1={1};
parameter __omit_DC1={1};
Eqn #() Eqn1 ();
Idc #(.I(I1)) I (.1(_net0),.2(0));
R #(.R(1)) R1 (.p(_net0),.n(0));
endmodule // main

